src/pymor/reductors/ph/ph_irka.py

```diff
@@ -80,11 +80,11 @@ def reduce(self, rom0_params, tol=1e-4, maxit=100, num_prev=1, projection='orth'
         rom
             Reduced-order |PHLTIModel|.
         """
-        one_sided_irka_reductor = OneSidedIRKAReductor(self.fom.to_lti(), 'V')
+        one_sided_irka_reductor = OneSidedIRKAReductor(self.fom.to_qless_model(), 'V')
         _ = one_sided_irka_reductor.reduce(rom0_params, tol=tol, maxit=maxit, num_prev=num_prev,
                                            projection=projection, conv_crit=conv_crit, compute_errors=compute_errors)
-
-        self._pg_reductor = PHLTIPGReductor(self.fom, one_sided_irka_reductor.V)
+        
+        self._pg_reductor = PHLTIPGReductor(self.fom.to_generalized_model(), one_sided_irka_reductor.V)
```

Comment:
I guess that the `to_generalized_model` method can be removed?

```suggestion
        self._pg_reductor = PHLTIPGReductor(self.fom, one_sided_irka_reductor.V)
```